src/styles/utile/_normalize.scss

```diff
@@ -0,0 +1,11 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
```

Comment:
i think it is _reset.scss

Reply:
Nope. See section nr 2. link: https://byby.dev/normalize-css

Reply:
oh thanks